packages/blade/src/components/Form/FormLabel.tsx

```diff
@@ -125,7 +136,8 @@ const FormLabel = ({
 
   const Component = as;
   // only set 120px label when device is desktop
-  const width = isLabelLeftPositioned && isDesktop ? makeSize(size[120]) : 'auto';
+  // const width = isLabelLeftPositioned && isDesktop ? makeSize(sizeToken[120]) : 'auto';
+  const width = isLabelLeftPositioned && isDesktop ? '176px' : 'auto'; // TODO: use size token
```

Comment:
Need to clarify this with design

Reply:
120px => 176px?? 

Won't this be breaking change? could cause layout breakages.